Swat/SwatFileEntry.php

```diff
@@ -680,16 +610,20 @@ private static function parseFileUploadSize($ini_value)
             switch ($size) {
                 case 'P':
                     $value *= 1024;
-                // no break
+
+                    // no break
```

Comment:
These comments are getting indented incorrectly.

Reply:
The indentation is correct (lined up with the rest of the lines inside the `case` as shown [here](https://www.php-fig.org/psr/psr-12/#52-switch-case)).  The extra line is that's being added before the comment is wrong, I agree.  I suspect it's because of the `blank_line_before_statement` rule that adds a line before each `case:`, but maybe it thinks the comment is part of the following block instead of the preceding?  I'm not sure, but I'm pushing a correction.

(In this case, we could probably just switch to a `match` anyway.)